Helper/Data.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+
+namespace MagentoHackathon\Toolbar\Helper;
+
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\State;
+use Magento\Store\Model\StoreManagerInterface;
+
+class Data extends AbstractHelper
+{
+    /**
+     * Get the url for this module
+     * @param  string $route
+     * @param  array $params
+     * @return string
+     */
+    public function getUrl($route = '', $params = [])
+    {
+        $route = 'hackathon_toolbar/' . $route;
+
+        return $this->_getUrl($route, $params);
+    }
+
+    /**
+     * Get a Config value
+     *
+     * @param $path
+     * @return mixed
+     */
+    public function getConfigValue($path)
+    {
+        return $this->scopeConfig->getValue($path);
+    }
+
+    /**
+     * Get the current Request
+     *
+     * @return \Magento\Framework\App\RequestInterface
+     */
+    public function getRequest()
+    {
+        return $this->_getRequest();
```

Comment:
Hmm yeah, bit perhaps cleaner to move the methods using the request to this helper. Already did that for a few, to simplify the toolbar itself.